doc/en/getting-started.rst

```diff
@@ -19,17 +19,15 @@ Installation and Getting Started
 Installation
 ----------------------------------------
 
-Installation options::
+Installation::
 
-    pip install -U pytest # or
-    easy_install -U pytest
+    pip install -U pytest
 
 To check your installation has installed the correct version::
 
     $ pytest --version
     This is pytest version 3.0.0, imported from $PYTHON_PREFIX/lib/python3.5/site-packages/pytest.py
 
-If you get an error checkout :ref:`installation issues`.
```

Comment:
Does this refer to an existing document? If so, that reference seems useful to me?